CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
 - Use `TypeVar` for alternate constructors ([#983](https://github.com/stac-utils/pystac/pull/983))
 - Behavior when required fields are missing in `Item.from_dict` ([#994](https://github.com/stac-utils/pystac/pull/994))
 - By default, `ItemCollection` now clones items in iterator (`clone_items=True`) ([#1016](https://github.com/stac-utils/pystac/pull/1016))
+- Update links in doc notebooks to not point to specific versions ([#1039](https://github.com/stac-utils/pystac/pull/1039))
```

Comment:
CHANGELOG entry is is in the incorrect section (should be in `[Unreleased]`).

Reply:
:facepalm: well that certainly makes sense...